staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -117,10 +115,10 @@ func (c *converterImpl) FromUnstructured(u map[string]interface{}, obj interface
 		newObj := reflect.New(t.Elem()).Interface()
 		newErr := fromUnstructuredViaJSON(u, newObj)
 		if (err != nil) != (newErr != nil) {
-			glog.Fatalf("FromUnstructured unexpected error for %v: error: %v", u, err)
+			panic(fmt.Errorf("FromUnstructured unexpected error for %v: error: %v / %v", u, err, newErr))
```

Comment:
If there is no difference why not use a built-in construct? :) Panic also prints the stacktrace.
When you run your unit/integration tests you want all of them to finish even if there is an assertion failure. You don't want a library function ending the whole test run. Don't see why Fatalf might be better in this case.